lib/pardot/api_builder.rb

```diff
@@ -48,13 +48,17 @@ def get(path, params = {}, result = results_field_name)
     end
 
     def post(path, params = {}, result = results_field_name)
-      response = client.post results_field_name, path, {}, 0, params
```

Comment:
I believe this would break the posting of the params, unless something in the gem has changed significantly.
The post didn't pass the parameters prior to https://github.com/hedgeyedev/ruby-pardot/commit/9c74f309e255974eda8ad4771c559cec9242ecd0

Reply:
Thanks for the update! That makes a lot more sense and is what the API docs describe. I suspect the upstream developers put the params in the query string to make it easier to test? There's also a [small limitation in the fakeweb gem for POST body](https://github.com/chrisk/fakeweb#known-issues-).
I'll see if I can change the specs to reflect the correct behavior.